daml-lf/spec/daml-lf-1.rst

```diff
@@ -235,8 +235,8 @@ Version: 1.12
     details
 
 
-Version: 1.13 (preview)
-.......................
+Version: 1.13
```

Comment:
You might want to copy the changelog from 1.dev. `A BigNumeric` is a bit brief and also seems like a typo.